styles/styles.css

```diff
@@ -1,8 +1,264 @@
-/* 
- * Put project specific base styles here.
```

Comment:
@karri-sek , can you please check if we need this whole file to be included in abp ? Shouldn't it be coming from milo already ?